src/BlockTypes/ProductSearch.php

```diff
@@ -124,4 +124,40 @@ protected function render( $attributes, $content ) {
 			$label_markup . $field_markup
 		);
 	}
+
+	/**
+	 * Extra data passed through from server to client for block.
+	 *
+	 * @param array $attributes  Any attributes that currently are available from the block.
+	 *                           Note, this will be empty in the editor context when the block is
+	 *                           not in the post content on editor load.
+	 */
+	protected function enqueue_data( array $attributes = [] ) {
+		parent::enqueue_data( $attributes );
+
+		$gutenberg_version = '';
+
+		if ( is_plugin_active( 'gutenberg/gutenberg.php' ) ) {
+			if ( defined( 'GUTENBERG_VERSION' ) ) {
+				$gutenberg_version = GUTENBERG_VERSION;
+			}
+			if ( ! $gutenberg_version ) {
+				$gutenberg_data    = get_file_data(
+					WP_PLUGIN_DIR . '/gutenberg/gutenberg.php',
+					array( 'Version' => 'Version' )
+				);
+				$gutenberg_version = $gutenberg_data['Version'];
+			}
+		}
+
+		$this->asset_data_registry->add(
+			'isBlockVariationAvailable',
+			version_compare( get_bloginfo( 'version' ), '6.1', '>=' ) || version_compare( $gutenberg_version, '13.4', '>=' )
+		);
+
+		/**
+		 * Fires after cart block data is registered.
+		 */
+		do_action( 'woocommerce_blocks_cart_enqueue_data' );
```

Comment:
I'm not sure to understand why this action needs to be fired. Is it an overview or it's related to the Product Search somehow?

Reply:
I'm sorry! It's a copypasta issue.